### PR TITLE
Add githubteacher San Diego file

### LINKED ISSUE #21: Create a new file for githubteacher San Diego
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Discuss
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by githubteacher**
closes #21

### CODE REVIEW

**Comment by goodysinclair** [comment]
Buy a mop.

File: githubteacher-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# San Diego, CA
+
+### Recommendations
+- Eat at City Tacos!
+
+### Fun Facts
+- Apparently my neighborhood is flooding at this moment.
```
Comment (goodysinclair) [inline githubteacher-sandiego.md]: dam 💧

**Review by goodysinclair — APPROVED** [review]
Heroes need no approval.

**Review by millionsm1 — APPROVED** [review]

**Review by hectorsector — APPROVED** [review]
👍 but I hear heroes need no approval

**Pull request merged**